charts/victoria-metrics-operator/values.yaml

```diff
@@ -30,6 +30,8 @@ image:
   pullPolicy: IfNotPresent
 
 crds:
+  # -- manages CRD creation. Disables CRD creation only in combination with `crds.plain: false` due to helm dependency conditions limitation
```

Comment:
Why not have both `crds.plain` and `crds.enabled` as the conditions of dependency `crds`?
```
  - name: crds
    version: "0.0.*"
    condition: crds.plain, crds.enabled
```

Reply:
ohh, nvm, it's "or" combination.